### ISSUE TITLE
error on atlassian.net: StorageConsent is not defined

### ISSUE THREAD

**Issue by neeasade** [qutebrowser/qutebrowser#7703]
**Version info**:

qutebrowser v2.5.3
Git commit: 2fe78b764 on HEAD (2023-02-17 11:12:46 +0100)
Backend: QtWebEngine 5.15.2, based on Chromium 83.0.4103.122
Qt: 5.15.2

CPython: 3.10.10
PyQt: 5.15.6

sip: 6.5.1
colorama: 0.4.4
jinja2: 3.1.1
pygments: 2.11.2
yaml: 6.0
adblock: 0.5.2
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebEngine: 5.15.5
PyQt5.QtWebKitWidgets: no
pdf.js: 3.3.122 (bundled)
sqlite: 3.33.0
QtNetwork SSL: Secure Transport, macOS 10.16

Style: QMacStyle
Platform plugin: cocoa
OpenGL: None
Platform: macOS-10.16-x86_64-i386-64bit, 64bit
Frozen: True
Imported from /Applications/qutebrowser.app/Contents/MacOS/qutebrowser
Using Python from /Applications/qutebrowser.app/Contents/MacOS/qutebrowser
Qt library executable path: /Applications/qutebrowser.app/Contents/MacOS/PyQt5/Qt5/libexec, data path: /Applications/qutebrowser.app/Contents/MacOS/PyQt5/Qt5
OS Version: 10.16, x86_64

**Does the bug happen if you start with `--temp-basedir`?**:

yes 

**Description**

Problem url being pulled in is: https://projects.invisionapp.com/measure/utilities.js, in the console (:devtools) I see: 

```
utilities.js?loadBraze=false:46 Uncaught ReferenceError: StorageConsent is not defined
    at initSegment (utilities.js?loadBraze=false:46)
    at utilities.js?loadBraze=false:54
    at initializeAnalyticsUI (utilities.js?loadBraze=false:54)
    at utilities.js?loadBraze=false:54
    at XMLHttpRequest.request.onreadystatechange (utilities.js?loadBraze=false:36)
```

And then I can't click things due to the uncaught error -- happens with `:set content.blocking.enabled` to true or false

❓ what I'm wondering is if this is a google chrome builtin or something 

**How to reproduce**

I encounter this on a jira board on atlassian.net -- making an issue to see if anyone else encounters this (couldn't find anything and could not reproduce on google chrome)

**Comment by The-Compiler** [comment]
Do you have a public link where I could reproduce this? Can you try Qt 6? (#7202)

**Comment by The-Compiler** [comment]
Closing as I don't believe this is a qutebrowser issue (might be in QtWebEngine?), and I won't be able to do anything without having a way to reproduce.

**Comment by neeasade** [comment]
FYI fixed with qb 3.0.0 (congrats on qt6 by default 😤 )
